forge-game/src/main/java/forge/game/combat/Combat.java

```diff
@@ -653,8 +653,7 @@ public final boolean removeAbsentCombatants() {
         CardCollection missingCombatants = new CardCollection();
         for (Entry<GameEntity, AttackingBand> ee : attackedByBands.entries()) {
             CardCollectionView atk = ee.getValue().getAttackers();
-            for (int i = atk.size() - 1; i >= 0; i--) { // might remove items from collection, so no iterators
-                Card c = atk.get(i);
+            for (Card c : atk) {
```

Comment:
How about this:

`for (Card c : ee.getValue().getAttackers()) {`

Reply:
Yep, agreed :)